internal/hcs/service.go

```diff
@@ -0,0 +1,39 @@
+//go:build windows
+
+package hcs
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/Microsoft/hcsshim/internal/interop"
+	hcsschema "github.com/Microsoft/hcsshim/internal/schema2"
+)
+
+// GetServiceProperties returns properties of the host compute service.
+func GetServiceProperties(ctx context.Context, q hcsschema.PropertyQuery) (properties *hcsschema.ServiceProperties, err error) {

```

Comment:
>q [](http://example.com/codeflow?start=47&length=1)

since GetServiceProperties accept an array of property types query, can we use the same convention as System.Properties() that accept a set of PropertyType and construct PropertyQuery inside this function

Reply:
i picked this up from master branch. changing this would cause merge issues during upstream. suggest leave as is. thoughts? 
https://github.com/microsoft/hcsshim/blob/ba4bfca3fdbe5421395db4a5d65788646c780adf/internal/hcs/service.go

Reply:
ok, keep it align to master branch